### PR TITLE
Sloeber fails to switch workspaces #935

### PR DESCRIPTION
**Pull request by wimjongman**
Added logging
Moved to Tycho 1.2.0
Moved to Photon
Changed parent name to sloeber

### CODE REVIEW

**Comment by wimjongman** [comment]
@jantje Ihave change the parent name to sloeber.io instead of brodykenrick. 

I suspect there are some maven central keys associated with this name and repo? Pushing to maven central fails.

**Comment by wimjongman** [comment]
Also solves issue 641

File: readme.md [hunk 1]
```diff
@@ -63,7 +63,8 @@ Mac OSX and Linux
 
 You can control the maven build with the following profiles: (this list may not be complete as new eclipse versions are added nearly immediately)
 
-* oxygen
+* photo (builds against the photon repositories (4.8))
```
Comment (uzi18) [inline readme.md:66]: typo?

Comment (jantje) [reply readme.md:66]: This is old by now

**Comment by jantje** [comment]
@wimjongman 
With travis out of the way
I replaced com.github.brodykenrick.arduino-eclipse-plugin with io.sloeber.arduino-eclipse-plugin
I added log4j
I also removed mylyn as it starts to have problems due to lack of contributors. see  https://bugs.eclipse.org/bugs/show_bug.cgi?id=569078#c4 for more info

I think these were all the changes this pull request contained so I think we can close it.

**Comment by wimjongman** [comment]
Cheers!

**Pull request closed without merging**